src/core/ext/client_config/client_channel.c

```diff
@@ -173,20 +176,26 @@ static void on_resolver_result_changed(grpc_exec_ctx *exec_ctx, void *arg,
   grpc_error *state_error = GRPC_ERROR_CREATE("No load balancing policy");
 
   if (chand->resolver_result != NULL) {
-    lb_policy = grpc_resolver_result_get_lb_policy(chand->resolver_result);
+    grpc_lb_policy_args lb_policy_args;
+    lb_policy_args.addresses =
+        grpc_resolver_result_get_addresses(chand->resolver_result);
```

Comment:
why not inline the function? It seems it's only used here and it's a one-liner. Same for `grpc_resolver_result_get_lb_policy_args` and `grpc_resolver_result_get_lb_policy_name` below.